www/index.html

```diff
@@ -72,6 +72,9 @@ <h3>Supported Tile Sources</h3>
             <li><a href='/examples/tilesource-custom/'>
                 <span>Custom Tile Sources</span>
             </a></li>
+            <li><a href='/examples/tilesource-iris/'>
+                <span>IRIS</span>
```

Comment:
It looks like you're using the capitalization `Iris` elsewhere... is that correct? If so, please use that here as well for consistency.